openwisp_firmware_upgrader/tests/test_openwrt_upgrader.py

```
@@ -42,7 +43,9 @@ def mocked_exec_upgrade_not_needed(command, exit_codes=None):
     return cases[command]
 
 
-def mocked_exec_upgrade_success(command, exit_codes=None, timeout=None):
+def mocked_exec_upgrade_success(
+    command, exit_codes=None, timeout=None, raise_unexpected_exit=None
```

Ruff (0.14.14)

Unused function argument: `exit_codes`

(ARG001)